packages/odyssey-storybook/src/components/odyssey-mui/Form/Form.stories.tsx

```diff
@@ -287,10 +287,10 @@ export const KitchenSink: StoryObj<FormProps> = {
           description="This information will be used to track your passengers' whereabouts."
           alert={
             <Infobox severity="error" role="alert" title="Standby for boarding">
-              <Typography paragraph>
```

Comment:
This is hard to see. I'm really thinking we should name this `Paragraph`. Since we own them, it makes sense.